hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestStorageContainerManager.java

```diff
@@ -593,7 +591,7 @@ public void testCloseContainerCommandOnRestart() throws Exception {
           new TestStorageContainerManagerHelper(cluster, conf);
 
       helper.createKeys(10, 4096);
-      Thread.sleep(5000);
+      Thread.sleep(10000);
```

Comment:
Repeated check (eg. `GenericTestUtils.waitFor`) should be preferred over fixed length sleep:

1. allow quicker test execution if possible
2. explicit condition documents what the test is waiting for

Reply:
done

Reply:
Thanks.  What about the other sleep?

Reply:
I spent some time looking around in the code. Not sure how we can do this cleanly for the other sleep.

Reply:
Changed this to ->
 - waiting till the replication manager comes up using GenericTestUtils.waitFor and then
 - wait some more to give it enough time to process containers